src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +242,58 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| Myelin water fraction map                    | MWFmap    | In percentage (%). MWF maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Macromolecular tissue volume (MTV) map       | MTVmap    | In arbitrary units (a.u.). MTV maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Combined PD/T2 map                           | PDT2map   | In arbitrary units (a.u.). Combined PD/T2 maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Quantitative susceptibility map (QSM)        | Chimap    | In parts per million (ppm). QSM allows for determining the underlying magnetic susceptibility of tissue (Chi) ([Wang & Liu, 2014](https://onlinelibrary.wiley.com/doi/10.1002/mrm.25358)). Chi maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| RF transmit field map                        | TB1map    | In percent units (p.u.). Radio frequency (RF) transmit field maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Given where we ended up with arbitrary units (`arbitrary` instead of `a.u.`), I don't think `p.u.` is the abbreviation we'll want for this. Also, does that refer to percentage points, percentage, or some other unit I'm not familiar with?

Reply:
Naming this one is also a bit tricky. B1+ is commonly defined as a  normalized multiplicative factor such that: FA_actual = **B1+** * FA_nominal.  Maybe we can switch the unit description to `arbitrary` but add this in the description. Happy to do the same for `%`, as @tiborauer pointed out, these are not absolute units. 

@ChristophePhillips do you follow the `FA_actual = B1+ * FA_nominal` for `B1+` as well? IIRC, `p.u.` was your suggestion.

Reply:
> I think it will require a unit if the values are going to be validated automatically at some point.

@tsalo @tiborauer looks like validation of the range of images at arbitrary units is more of a future discussion. I will set the units as arbitrary and will RECOMMEND corresponding range adjustments in the explanation for now. How does that sound?

Reply:
It may be worth trying to support ranges in the schema, so I agree that this can wait and go in a new issue.

Reply:
6f5cf15